pkgs/tools/system/kanata/default.nix

```diff
@@ -8,16 +8,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "kanata";
-  version = "1.5.0";
+  version = "1.6.0";
 
   src = fetchFromGitHub {
     owner = "jtroo";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-ci/0Ksmi0uNHIvpZlihWvGeNabzmk+k3fUeuMDVpFeE=";
+    sha256 = "sha256-UZ6WoUZcnqN9k277ikix91UTnqQsUgZW69vbSNNbyHU=";
   };
 
-  cargoHash = "sha256-IzgVF6SHJjOB48VehQ5taD5iWQXFKLcVBWTEl3ArkGQ=";
+  cargoHash = "sha256-DLOQnQJtrR7A/elOTVXEnIdLM/FuWYWAPqpbSAXHDMo=";
```

Comment:
Darwin builds are failing due to this hash. Needs investigation.

Reply:
@bmanuel Any updates?

This hash is correct for linux systems but wrong for darwin systems.  Any ideas from @NixOS/darwin-maintainers ?

Reply:
Can you make the hash conditional on platform (e.g., have a different one for Darwin or Linux)?

Reply:
> Can you make the hash conditional on platform (e.g., have different ones for Darwin and Linux)?

Done.

However, it seems a bit weird since there is only [one other package][1] in Nixpkgs doing so.

[1]: https://github.com/NixOS/nixpkgs/blob/f136c5224d4f3142a8b7fe3ed6de7b0638e161c1/pkgs/by-name/at/atuin/package.nix#L22-L26

Reply:
If the Rust team approves the pattern, we could unblock some PRs made by r-ryantm, for example #276488.

Reply:
This seems to be caused by the `native-windows-gui` package having both `README.md` and `readme.md`:

```
❯ tar tf $(nix-build -A kanata.cargoDeps) | fgrep kanata-1.6.0-vendor.tar.gz/native-windows-gui/ | grep -i readme.md
kanata-1.6.0-vendor.tar.gz/native-windows-gui/README.md
kanata-1.6.0-vendor.tar.gz/native-windows-gui/readme.md
```

Reply:
So it's due to case-insensitivity of the file system again. That does suggest an alternative solution by renaming/removing one of those files in postFetch. Then Linux and Darwin should agree on the hash again.

Technically this would be a broader fix, the problem isn't Darwin per se, it's the case-insensitivity and you can run a case-insensitive file system on Linux just as well.

Reply:
I created https://github.com/NixOS/nixpkgs/issues/308089 for this.